app/views/landing-pages/roblox/PageRoblox.vue

```diff
@@ -229,7 +230,7 @@ export default {
       i18nData,
       videoId,
       youtubeId,
-      thumbnailUrl: `https://videodelivery.net/${videoId}/thumbnails/thumbnail.jpg?time=3.000s`
+      thumbnailUrl: `https://videodelivery.net/${videoId}/thumbnails/thumbnail.jpg?time=0.000s`
```

Comment:
Let's try a thumbnail from later in the video:
```suggestion
      thumbnailUrl: `https://videodelivery.net/${videoId}/thumbnails/thumbnail.jpg?time=11.000s`
```